config/manager/manager.yaml

```diff
@@ -1,15 +1,8 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    control-plane: controller-manager
-  name: system
----
```

Comment:
since the namespace is now defined outside of the `manager` dir does that cause any problems?

Reply:
Like what?

Reply:
no idea 😄 but I noticed this is done elsewhere, so I don't think it matters

Reply:
:D Nah, it's good, because the main prepare manifest creates it. The only thing could be if it's applied without a kustomize I guess, but that's the approach flux uses as well.